supervision/detection/tools/polygon_zone.py

```diff
@@ -20,21 +21,29 @@ class PolygonZone:
         polygon (np.ndarray): A polygon represented by a numpy array of shape
             `(N, 2)`, containing the `x`, `y` coordinates of the points.
         frame_resolution_wh (Tuple[int, int]): The frame resolution (width, height)
-        triggering_position (Position): The position within the bounding
-            box that triggers the zone (default: Position.BOTTOM_CENTER)
+        triggering_anchors (Iterable[sv.Position]): A list of positions specifying
+            which anchors of the detections bounding box to consider when deciding on
+            whether the detection fits within the PolygonZone
+            (default: (sv.Position.BOTTOM_CENTER,)).
         current_count (int): The current count of detected objects within the zone
         mask (np.ndarray): The 2D bool mask for the polygon zone
     """
 
+    @deprecated_parameter(
```

Comment:
I'd love to have a way to attach a custom message, for example, mentioning the version of the `supervision` package when support for deprecated parameter names will be removed.

Reply:
Thansk, done. Please have a look: https://github.com/roboflow/supervision/pull/910/files#diff-b15cd2cdcbbed3ddb5617e22208f7e084b583346acaf59ca8b9dcbbf7e85a49bR10

Let me know what you think about this solution.

Reply:
We usually add a message like this: `Color.white()` is deprecated and will be removed in `supervision-0.22.0`. Use `Color.WHITE` instead.` Could we add deprecation drop version as well?